.github/dependabot.yml

```diff
@@ -32,8 +32,6 @@ updates:
       prefix: "GitHub Action"
     labels:
       - "type:dependencies"
-    reviewers:

```

Comment:
Taking the reviewers will reduce the number of notifications. Is there a way to notify reviewer if the CI fails so that manual action can be taken?

Reply:
There might be a better way, but we should be able to find them quick enough by checking for failed PRs from dependabot or uefibot.